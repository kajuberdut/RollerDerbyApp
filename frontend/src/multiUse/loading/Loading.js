import React from "react";
import "./Loading.css";

/**  
* Display is loading page
*/

function Loading() {
    return (
        <div data-testid="loading-indicator" className="Loading"></div>
    );
  }
  

export default Loading; 