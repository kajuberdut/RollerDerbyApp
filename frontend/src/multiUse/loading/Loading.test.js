import React from "react";
import { render } from "@testing-library/react";
import Loading from "./Loading";
import { BrowserRouter } from "react-router-dom";


describe("Loading tests", () => {

    test("renders loading page", () => {
      localStorage.setItem("user", null);
      render(
        <BrowserRouter>
              <Loading />
        </BrowserRouter>,
      );
    
    }); 
  
    it("matches snapshot", function() {
      localStorage.setItem("user", null);
      const { asFragment } = render(
        <BrowserRouter>
              <Loading />
        </BrowserRouter>,
      );
        expect(asFragment()).toMatchSnapshot();
    });

})
