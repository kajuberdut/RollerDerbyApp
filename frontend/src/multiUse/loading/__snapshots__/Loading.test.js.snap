// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading tests matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Loading"
    data-testid="loading-indicator"
  />
</DocumentFragment>
`;
